sunpy/sun/models.py

```diff
@@ -198,3 +208,43 @@ def differential_rotation(duration: u.s, latitude: u.deg, *, model='howard', fra
         rotation -= 0.9856 * u.deg / u.day * duration
 
     return Longitude(rotation.to(u.deg))
+
+
+
+_MODEL_DATA_DIR = pathlib.Path(__file__).parent.absolute()/"data"
+
+# Dictionary mapping model names to file paths
+_CHROMOSPHERE_MODELS = {
+    "avrett_loeser_2008": _MODEL_DATA_DIR / "chromosphere_avrett_Loeser_2008_model.ecsv"
+}
+
+# Cache to store loaded models
+_chromosphere_cache = {}
+
+def _read_chromosphere_data(model_name):
+    """Reads the specified chromosphere model data."""
+    return QTable.read(_CHROMOSPHERE_MODELS[model_name], format="ascii.ecsv")
+
+def __getattr__(name):
+    """
+    Provides dynamic access to chromosphere models.
+
+    Parameters
+    ----------
+    name : str
+        The requested model name.
+
+    Returns
+    -------
+    QTable
+        The cached chromosphere model data.
+    """
+
+    if name.startswith("chromosphere_"):
+        model_name = name.replace("chromosphere_", "")
+        if model_name in _CHROMOSPHERE_MODELS:
+            if model_name not in _chromosphere_cache:
+                _chromosphere_cache[model_name] = _read_chromosphere_data(model_name)
+            return _chromosphere_cache[model_name]
```

Comment:
This should just just check for the model and return it, else error or return I assume the other items in this module.